### PR TITLE
feat: Fixed lshr and shl being swapped in PrettyEDSL

### PR DESCRIPTION
**Pull request by lfrenot**

### CODE REVIEW

File: SSA/Projects/InstCombine/scripts/test/minimal.lean [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import SSA.Projects.InstCombine.LLVM.Semantics
```
Comment (tobiasgrosser) [inline SSA/Projects/InstCombine/scripts/test/minimal.lean]: This file seems unrelated. Also, do you need to update the test cases for this specific change?

Comment (lfrenot) [reply SSA/Projects/InstCombine/scripts/test/minimal.lean]: Oh, yes I didn't mean to have this file still there

Comment (lfrenot) [reply SSA/Projects/InstCombine/scripts/test/minimal.lean]: As for updating the test cases, I wanted to do some edits to the generators first
